tests/testthat/test-downloadModule.R

```diff
@@ -45,6 +45,7 @@ test_that("downloadModule downloads and unzips a parent module", {
   d_expected <- moduleMetadata("LCC2005", tmpdir)$childModules %>%
     c(m, "data", "testthat") %>% sort()
 
+#   expect_equal(length(f), 42)
   expect_equal(length(f), 44)
```

Comment:
Interesting. Did u or Eliot make some changes in LLC2005?

Yong Luo

> On Feb 2, 2016, at 11:17 PM, Alex Chubaty <notifications@github.com> wrote:
> 
> In tests/testthat/test-downloadModule.R:
> 
> > @@ -45,6 +45,7 @@ test_that("downloadModule downloads and unzips a parent module", {
> >    d_expected <- moduleMetadata("LCC2005", tmpdir)$childModules %>%
> >      c(m, "data", "testthat") %>% sort()
> >  
> > +#   expect_equal(length(f), 42)
> >    expect_equal(length(f), 44)
> No it can't be because I haven't changed the test and it passes on other machines including travis.
> 
> —
> Reply to this email directly or view it on GitHub.
>